test/spec/features/dragging/HoverFixSpec.js

```diff
@@ -77,6 +77,31 @@ describe('features/dragging - HoverFix', function() {
       }
     ));
 
+
+    it('should stop propagation', inject(
```

Comment:
This test case does not necessarily test the order in which the events are fired. You can check out the commit that I planned to add on top before I noticed that review was redone ;)
https://github.com/bpmn-io/diagram-js/compare/fixup-create...fixup-create-test?expand=1

Reply:
Thanks, just modified the test case with your bonus material 👍